backend/src/server/routes/user_tag.go

```diff
@@ -14,4 +14,7 @@ func UserTag(usersRouter fiber.Router, userTagService services.UserTagServiceInt
 
 	userTags.Post("/", userTagController.CreateUserTags)
 	userTags.Get("/", userTagController.GetUserTags)
+
+	tagID := userTags.Group("/:tagID")
+	tagID.Delete("/", userTagController.DeleteUserTag)
```

Comment:
```suggestion
	tagID.Delete("/:tagID", userTagController.DeleteUserTag)
```

Reply:
Removing this alters the routing structure we use throughout the repo